cg/apps/lims/orderform.py

```diff
@@ -191,6 +192,7 @@ def parse_sample(raw_sample):
     sample = {
         "application": raw_sample["UDF/Sequencing Analysis"],
         "capture_kit": raw_sample.get("UDF/Capture Library version"),
+        "bait_set": db.latest_bed_version(raw_sample.get("UDF/Capture Library version")).shortname,
```

Comment:
I don't think this logic should go here

Reply:
this logic is for parsing samples from the orderform

Reply:
Let's look at this together when @mayabrandi is back on site